02-technical-guides/hierarchies/main.ipynb

```diff
@@ -0,0 +1,2608 @@
+{
```

Comment:
<p>Either here, or at the top, it might be nice to add an aside to check out the Table creation notebook as a pre-cursor. Just a suggestion, not required.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/atoti/atoti/pull/877/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='77'/>

Reply:
Added the note up top (since it seems more apt to leave the note up top where it is most likely to be seen) — lemme know what you think!